lux-lib/src/operations/sync.rs

```diff
@@ -267,6 +268,8 @@ async fn do_sync(
         project_lockfile.sync(dest_lockfile.local_pkg_lock(), lock_type);
     }
 
+    luarc::update_luarc(args.config);
```

Comment:
todo: move this to the `sync_dependencies` and `sync_test_dependencies` methods. We don't need to update the luarc when syncing build dependencies.